spec/support/work_mapping_fixtures.rb

```diff
@@ -10,7 +10,8 @@ def new_work_form_fixture
       abstract: abstract_fixture,
       related_links_attributes: related_links_fixture,
       license: license_fixture,
-      collection_druid: collection_druid_fixture
+      collection_druid: collection_druid_fixture,
+      publication_date: DateForm.new(year: '2024', month: '12')
```

Comment:
Should this be `publication_date_attributes` like the other nested fields?

Reply:
Could be. I changed it.